pkgs/by-name/wa/waydroid-helper/package.nix

```diff
@@ -0,0 +1,94 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  desktop-file-utils,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  libxml2,
+  python3Packages,
+  appstream,
+  glib,
+  cmake,
+  dbus,
+  systemd,
+  bash,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "waydroid-helper";
+  version = "0.1.2";
+  pyproject = false; # uses meson
+
+  src = fetchFromGitHub {
+    owner = "ayasa520";
+    repo = "waydroid-helper";
+    tag = "v${version}";
+    hash = "sha256-dYduO5Wi8Ia/pR1xQKPhC6Ek/1Q9fm2RaVuhm9KYiU0=";
+  };
+
+  postPatch = ''
+    substituteInPlace dbus/meson.build \
+      --replace-fail "dbus_policy_dir," "'${placeholder "out"}/share/dbus-1/system.d'," \
+      --replace-fail "dbus_service_dir," "'${placeholder "out"}/share/dbus-1/system-services',"
+    substituteInPlace systemd/meson.build \
+      --replace-fail ": systemd_system_unit_dir" ": '${placeholder "out"}/lib/systemd/system'" \
+      --replace-fail ": systemd_user_unit_dir" ": '${placeholder "out"}/lib/sysusers.d'"
+    sed -i '/test(/{N;/Validate appstream file/!b;:a;N;/)/!ba;d}' data/meson.build
```

Comment:
It could be helpful if you added a comment about why this is being done/what this does.